config/settings.py

```
@@ -5,17 +5,47 @@
 
+def get_django_forms_templates_path() -> str | None:
+    """Finds the path to the Django forms templates directory in site-packages.
+
+    This function searches through all known site-packages directories to locate the directory containing Django's form templates.
+    Falls back to searching sys.path for site-packages if not found.
+
+    Returns:
+        str or None: The path to the forms templates directory if found, otherwise None.
+    """
+    # Try all site-packages directories from the site module
+    for site_dir in site.getsitepackages() + [site.getusersitepackages()]:
```

Ruff (0.11.9)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)